packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -418,7 +418,7 @@ class CloudFormation {
 
                     if (completeErr) {
                       await this.collectStackErrors(cfnParentStackParams.StackName).then((errorDetails) => {
-                        completeErr.message = `${completeErr.message} \n ${errorDetails}`;
+                        completeErr.message = `${completeErr.message}${EOL}${errorDetails}`;
```

Comment:
This is going to impact telemetry. Please don't push this change.

Reply:
Won't it impact telemetry positively? Because you'll receive error messages that include the root cause instead of the umbrella error message?

Reply:
Because then we won't be able to group/bucketize issues at all. This information is already emitted in the telemetry in the `error_details` section

Reply:
Updated to use `printAmplifyException` instead. 

New screenshot attached to the PR description.